jetty-util/src/main/java/org/eclipse/jetty/util/ssl/KeyStoreScanner.java

```diff
@@ -43,6 +43,11 @@ public class KeyStoreScanner extends ContainerLifeCycle implements Scanner.Discr
     private final Scanner _scanner;
 
     public KeyStoreScanner(SslContextFactory sslContextFactory)
+    {
+        this(sslContextFactory, false);
```

Comment:
The default should be `true` because that was the default behavior prior to this change.